### ISSUE TITLE
Select currency per event, expense, ticket & sponsor

### ISSUE THREAD

**Issue by bmann** [opencollective/opencollective#1182]
My organization seems to be defaulted to USD. We do events all over the world, and ideally we would select local currency for each event.

This also means that invoices (for example) need to be able to select what currency they were in.

**Comment by piamancini** [comment]
Hi @bmann your events will be defaulted to your collective's currency. We don't support multicurrency collectives yet.

**Comment by BenJam** [comment]
++ 

it would seem useful to have this on record to save costs on transactions involving an exchange too, i.e. when a host has an account in currency X they would probably want to pay any expenses in currency X from that account/service to avoid needless FX rates.

**Comment by piamancini** [comment]
While we solve this (it will be solved with multicurrency) the work around @bmann is to do different collectives for the events you are hosting, each in its own currency, 
If you have set up an organization already, I can create activate the /apply so you can host them all in your organization like wwcode for example

**Comment by dodgepong** [comment]
Would it be possible for individuals to be able to submit expenses as any currency and have it automatically converted to the collective's currency according to current exchange rates (shown live to the user as they are submitting an expense)?

At the very least, could the "Amount" box in the expenses form indicate the collective's currency? `$` is 't sufficient due to multiple countries using different "dollars" as their currency (USD, CAD, NZD, etc.).

**Comment by piamancini** [comment]
yes, that seems like the best solution. This entails changing the way we handle currencies atm so it's not trivial.
